manifests/init.pp

```diff
@@ -32,7 +32,11 @@
   Hash $lvs_virtual_server = {},
 ) inherits keepalived::params {
 
-  class { 'keepalived::install': }
-  -> class { 'keepalived::config': }
-  -> class { 'keepalived::service': }
+  contain keepalived::install
+  contain keepalived::config
+  contain keepalived::service
+
+  Class['Keepalived::Install']
```

Comment:
Why the capitalisation here?  Perhaps it works because it's non case-sensitive???